lib/presentation/widget/visibility/visibility_list_image.dart

```diff
@@ -0,0 +1,52 @@
+import 'package:flutter/material.dart';
+
+import '../../../core/util/dimensions.dart';
+
+import '../../../data/model/production_companies_model.dart';
+import '../show_image.dart';
+
+class VisibilityListImage extends StatelessWidget {
+  const VisibilityListImage({
+    required this.movieDetailList,
+    super.key,
+  });
+
+  final List<dynamic> movieDetailList;
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      constraints: const BoxConstraints(
+        maxWidth: Dimensions.visibilityListImageConstraintsMaxWidth,
+      ),
+      child: ListView.builder(
+        physics: const NeverScrollableScrollPhysics(),
+        shrinkWrap: true,
+        itemCount: movieDetailList.length,
+        itemBuilder: (
+          BuildContext context,
+          index,
+        ) {
+          return Padding(
+            padding: const EdgeInsets.all(
+              Dimensions.visibilityListImagePadding,
+            ),
+            child: (movieDetailList[index] is ProductionCompaniesModel)
+                ? (movieDetailList[index].logoPath != null)
+                    ? showImgIfExists(
+                        movieDetailList[index]!.logoPath,
+                        Dimensions.smallPosterSize,
+                      )
+                    : const Text('')
+                : (movieDetailList[index].posterPath != null)
+                    ? showImgIfExists(
+                        movieDetailList[index]!.posterPath,
+                        Dimensions.smallPosterSize,
+                      )
+                    : const Text(''),
```

Comment:
Why do you need this empty string Texts?

Reply:
We removed that file and use visibility_list_of_strings instead, we no longer use that portion of code.